scenario1/experiment_orchestrator.py

```diff
@@ -0,0 +1,246 @@
+import argparse
+from datetime import datetime
+import os
+import requests
+import subprocess
+import time
+import yaml
+from pathlib import Path
+from kubernetes import config
+from kubernetes.client import Configuration
+from kubernetes.client.api import core_v1_api
+import kubernetes.client as client
+from kubernetes.client.rest import ApiException
+from kubernetes.stream import stream
+from kr8s.objects import Pod
+
+def start_vllm_server_client(benchmark_config, benchmark_name, k_client, mode, model):
+    """Start vLLM server with config parameters"""
+    vllm_params = benchmark_config['vllm']
+
+    server_args = [model,
+        '--gpu-memory-utilization', str(vllm_params['gpu_memory_utilization']),
+        '--block-size', str(vllm_params['block_size']),
+        '--max-model-len', str(vllm_params['max_model_len']),
+        '--max-num-batched-tokens', str(vllm_params['max_num_batched_tokens']),
+        '--max-num-seqs', str(vllm_params['max_num_seqs']),
+        '--long-prefill-token-threshold', str(vllm_params['long_prefill_token_threshold']),
+        '--seed', str(vllm_params['seed'])
+    ]
+
+    if vllm_params.get('enable_prefix_caching'):
+        server_args.append('--enable-prefix-caching')
+
+    if vllm_params.get('disable_log_requests'):
+        server_args.append('--disable-log-requests')
+
+    client_args = [
+        f"""git clone https://github.com/inference-sim/vllm-data-collection
+cd vllm-data-collection/scenario1
+pip install -r requirements.txt
```

Comment:
There is no requirements.txt in `scenario1/`

Reply:
Added now